eng/tox/create_whl_for_dependencies.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
```

Comment:
`create_whl_for_dependencies` also installs.

- `install_dependencies.py`?
- `install_dependency_whls.py`?

I won't block the PR based on this name though 👍

Reply:
Renamed to create_dependencies_and_install.py to kind of mirror create_package_and_install.py